react/ui-components/src/molecules/stories/VerticalFilterCard.stories.js

```
@@ -0,0 +1,318 @@
+
+// Template for the story
+const Template = (args) => <FilterCard {...args} />;
+
+const genderOptions = [
+  { code: "M", name: "Male" },
+  { code: "F", name: "Female" },
+  { code: "O", name: "Others" },
+];
+
+const children = [
+  <LabelFieldPair vertical={true}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)